### DISCUSSION TITLE
How to write a quarto book in two different languages

### DISCUSSION THREAD

**Discussion by ddotta** [quarto-dev/quarto-cli#5119] [Q&A] [answered]
Hello,

I would like to write a quarto book that can display each chapter in 2 different languages.  
Is there already an example of a book that does this?
Or do you have a suggestion on how to do it?  

Thank you in advance for your help. 🙏

**Comment by cderv** [comment, marked as answer]
I believe the issue and discussion to follow is https://github.com/quarto-dev/quarto-cli/issues/275

There are already some informations there and it got a bit better with help of [Project Profile](https://quarto.org/docs/projects/profiles.html) but it still not complete. 

Please follow the issue for next work on this

**Comment by ddotta** [comment]
Thank you @cderv  for your answer 👍 and sorry I didn't see this discussion
